src/DynamoCoreWpf/ViewModels/Core/GalleryViewModel.cs

```diff
@@ -0,0 +1,145 @@
+﻿using Dynamo.Core;
+using Dynamo.Interfaces;
+using Dynamo.UI.Commands;
+using Dynamo.ViewModels;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Xml.Serialization;
+
+namespace Dynamo.Wpf.ViewModels.Core
+{
+    public class GalleryContent : NotificationObject
+    {
+        private string header;
+        public string Header { get; set; }
+
+        private string body;
+        public string Body { get; set; }
+
+        private string image;
+        public string Image { get; set; }
+
+        private bool isCurrent;
+        public bool IsCurrent
+        {
+            get
+            {
+                return isCurrent;
+            }
+            set
+            {
+                isCurrent = value;
+                RaisePropertyChanged("IsCurrent");
+            }
+        }
+    }
+
+    public class GalleryContents
+    {
+        public List<GalleryContent> GalleryUIContents { get; set; }
+
+        public static GalleryContents Load(string filePath)
+        {
+            var galleryContents = new GalleryContents();
+
+            if (string.IsNullOrEmpty(filePath) || (!File.Exists(filePath)))
+                return galleryContents;
+
+            try
+            {
+                var serializer = new XmlSerializer(typeof(GalleryContents));
+                using (var fs = new FileStream(filePath, FileMode.Open, FileAccess.Read))
+                {
+                    galleryContents = serializer.Deserialize(fs) as GalleryContents;
+                    fs.Close(); // Release file lock
+                }
+            }
+            catch (Exception) { }
+
+            return galleryContents;
+        }
+    }
+
+    public class GalleryViewModel: ViewModelBase
+    {
+        #region public members
+        public string CurrentImage { get { return currentContent.Image; } }
+        public string CurrentHeader { get { return currentContent.Header; } }
+        public string CurrentBody { get { return currentContent.Body; } }
+        public List<GalleryContent> Contents { get { return contents; } }
+        public DelegateCommand MoveNextCommand { get; set; }
+        public DelegateCommand MovePrevCommand { get; set; }
+        #endregion
+
+        public GalleryViewModel(DynamoViewModel dynamoViewModel) 
+        {          
+            IPathManager pathManager = dynamoViewModel.Model.PathManager;
+            var galleryFilePath = pathManager.GalleryFilePath;
+
+            if (File.Exists(galleryFilePath))
```

Comment:
I don't think we need to check here, since `GalleryContents.Load` has done all the checkings. Perhaps we can do this (I don't think we want the `contents` to be `null`, do we?):

```csharp
var pathManager = dynamoViewModel.Model.PathManager;
var galleryFilePath = pathManager.GalleryFilePath;
contents = GalleryContents.Load(galleryFilePath).GalleryUIContents;
```